api/src/main/java/se/fortnox/reactivewizard/CollectionOptions.java

```diff
@@ -69,4 +71,12 @@ public SortOrder getOrder() {
     public void setOrder(SortOrder order) {
         this.order = order;
     }
+
+    public boolean isLastRecord() {
+        return lastRecord;
+    }
+
+    public void setNotLastRecord() {
```

Comment:
If we in the future would like to make this a nullable property, then a setter taking a boolean would mean that the api would stay (almost) intact?